Debug.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+
+namespace CMP1903M_Assessment_1.Debugging
+{
+    static class Debug
+    {
+        public const ConsoleColor ErrorColour = ConsoleColor.Red;
+
+        /// <summary>
+        /// Writes a normal message to the console using default colours
+        /// </summary>
+        /// <param name="text">Text to write to the console</param>
+        public static void Log(string text)
+        {
+            Console.ResetColor();
+            Console.WriteLine(text);
+        }
+
+        /// <summary>
+        /// Writes a spooky scary error message to the console in RED!
+        /// </summary>
+        /// <param name="text">Text to write to the console</param>
+        public static void LogError(string text)
+        {
+            Console.ForegroundColor = ErrorColour;
+            Console.WriteLine(text);
+            Console.ResetColor();
+        }
+    }
+}
```

Comment:
This could be achieved using the exception class, but I understand you probably want to have an excuse to create a new class, so fair play.

Example:
```csharp 
throw new Exception("This is an error!");
```